packages/api/src/router/friend.ts

```diff
@@ -0,0 +1,75 @@
+import { z } from 'zod';
+import { createTRPCRouter, publicProcedure } from '../trpc';
+
+export const friendRouter = createTRPCRouter({
+  /**
+   * Get all friends
+   */
+  all: publicProcedure.query(({ ctx }) => {
```

Comment:
A bit nitpicky, but I would like a more specific name for these functions, such as naming this function something like "getAllFriends" instead of all.